eng/pipelines/dotnet-framework-wcf.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+pr:
+- master
+
+variables:
+- template: variables/common.yml
+- name: manifest
+  value: manifest.wcf.json
+- name: testCategory
+  value: WCF
+stages:
+- template: ../common/templates/stages/build-test-publish-repo.yml
+  parameters:
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      buildMatrixType: platformVersionedOs
+      customBuildLegGrouping: pr-build
```

Comment:
I suspect this isn't needed.  See my comment in the manifest.wcf.json file.

Reply:
Fixed in #330